### PR TITLE
Post to a URL including the branch name

### PR DESCRIPTION
**Pull request by mbland**
I still need to update the README, and the code is ripe for some refactoring probably, but this scheme does allow for a `branchInUrlPattern` configuration that will allow URLs such as `/foobar/v0.9.x`.

@msecret If you want to start the review, I'll get the README updated later. Also feel free to press me on some refactoring.

When this is done, I'll post this configuration and get pages-releases.18f.gov running.

cc: @maya @arowla @ccostino @ertzeid

### CODE REVIEW

File: lib/site-builder.js [hunk 1]
```diff
@@ -290,13 +303,18 @@ exports.launchBuilder = function(info, builderConfig, done) {
...
 
 exports.makeBuilderListener = function(webhook, builderConfig, done) {
-  var org = builderConfig.githubOrg || config.githubOrg;
-
-  webhook.on('refs/heads/' + builderConfig.branch, function(info) {
-    if (info.repository.organization === org) {
-      exports.launchBuilder(info, builderConfig, done);
-    }
-  });
+  var org = builderConfig.githubOrg || config.githubOrg,
```
Comment (msecret) [inline lib/site-builder.js:306]: While I'm usually a fan of using comma, separated var declarations, this one looks a little messy when they're functions. I'd suggest this:

``` js
var whatever = 'whatever',
    handler;

handler = function() {
...
```

**Comment by msecret** [comment]
LGTM

File: lib/site-builder.js (hunk 1 above)
Comment (mbland) [inline lib/site-builder.js:306]: Yeah, I agree. Thanks for the nudge. Done.

**Comment by mbland** [comment]
Thanks, @msecret! Want to do the honors (when Travis is done)?

**Pull request merged**